### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by AlexandreCo**
Hello,

this patch add inovalley kw9015b rain and temperature  wheater sensor and an other rf433 noname wheater station

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -56,6 +56,8 @@ add_executable(rtl_433
 	devices/lacrossews.c
 	devices/esperanza_ews.c
 	devices/efergy_e2_classic.c
+	devices/inovalley-kw9015b.c
```
Comment (robho) [inline src/CMakeLists.txt]: You should update src/Makefile.am also

**Comment by merbanan** [comment]
Can you submit sample data to the rtl_433_tests repository ? And fix the Makefile.am code ?

**Comment by rct** [comment]
Ugh.  Maybe it's just me.  Naming is very subjective, but  rfwireless as a device / file name?  

I understand it is a "no name" weather station, but the source file name "rfwireless" seems unnecessarily vague and confusing.  It looks like it is temperature only, not a full weather station or only temperature messages are being decoded for now.

How about "generic temperature sensor 1", or "generic weather station 1"?

This device might generate a number of false positives for anything that will put a 3 byte row into the bit buffer and has timing anywhere near close.

It would be helpful if:
* The source / callback were renamed.
* The source file contained a little more of a comment than "Unknown Temperature weather station".

Also for the other device, inovalley-kw9015b.c, change r_device->name from ""KW9015B Transmitter" to "Inovalley kw9015b rain and Temperature weather station" since that's what gets printed in the usage info.

**Comment by rct** [comment]
(btw, my comment is in the context of rtl_433 now decoding 38 wireless devices, so things need to be a bit clearer if possible.)

**Pull request merged**